src/controller/java/src/matter/onboardingpayload/OnboardingPayload.kt

```diff
@@ -115,15 +115,9 @@ class OnboardingPayload(
    */
   var setupPinCode: Long = 0
 ) {
-  var optionalQRCodeInfo: HashMap<Int, OptionalQRCodeInfo>
-  private val optionalVendorData: HashMap<Int, OptionalQRCodeInfo>
-  private val optionalExtensionData: HashMap<Int, OptionalQRCodeInfoExtension>
-
-  init {
-    optionalQRCodeInfo = HashMap()
-    optionalVendorData = HashMap()
-    optionalExtensionData = HashMap()
-  }
+  var optionalQRCodeInfo: HashMap<Int, OptionalQRCodeInfo> = HashMap()
+  val optionalVendorData: HashMap<Int, OptionalQRCodeInfo> = HashMap()
```

Comment:
We should mark optionalVendorData and optionalExtensionData as private since they are not accessible from outside the class.

Reply:
We are accessing optionalVendorData and optionalExtensionData in the "testQRCodeWithOptionalData" method. We should have some way to access this data from outside of the class.

Reply:
If the only reason for these fields to be public or internal is testing, it might be worth considering a refactor. Could we testing them indirectly instead of exposing internal details of the class

Reply:
I updated these fields to be private and we are now using the public getters that return these maps as lists.